test/pummel/test-hash-seed.js

```diff
@@ -1,31 +1,25 @@
 'use strict';
 
 // Check that spawn child doesn't create duplicated entries
-require('../common');
-const Countdown = require('../common/countdown');
+const common = require('../common');
 const REPETITIONS = 2;
 const assert = require('assert');
 const fixtures = require('../common/fixtures');
-const { spawn } = require('child_process');
+const { promisify } = require('util');
+const { execFile } = require('child_process');
+const execFilePromise = promisify(execFile);
 const targetScript = fixtures.path('guess-hash-seed.js');
-const seeds = [];
 
-const requiredCallback = () => {
+const requiredCallback = common.mustCall((results) => {
+  const seeds = results.map((val) => val.stdout.trim());
   console.log(`Seeds: ${seeds}`);
```

Comment:
If the console.log is not part of the actual test... can we convert this into a debuglog instead?

At the top of the file:

```js
const debug = require('util').debuglog('test');
```

Then here..

```js
debug(`Seeds: ${seeds}`);
```

Reply:
Sure, done. (`util` was already being imported so not exactly this way, but done.)